### PR TITLE
🎉 source: clickhouse setup docs

### PR DESCRIPTION
**Pull request by subodh1810**
## What
Issue : https://github.com/airbytehq/airbyte/issues/3317

Tested it in local
Screenshot 2021-05-12 at 4 45 57 PM

## How
*Describe the solution*

## Pre-merge Checklist
- [x] *Run integration tests*
- [x] *Publish Docker images*

## Recommended reading order
1. `test.java`
1. `component.ts`
1. the rest

### CODE REVIEW

File: airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/bad83517-5e54-4a3d-9b53-63e85fbd4d7c.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "sourceDefinitionId": "bad83517-5e54-4a3d-9b53-63e85fbd4d7c",
+  "name": "ClickHouse",
+  "dockerRepository": "airbyte/source-clickhouse./gradlew generateSeed./gradlew generateSeed",
```
Comment (ChristopheDuong) [inline airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/bad83517-5e54-4a3d-9b53-63e85fbd4d7c.json:4]: This is bad

Comment (subodh1810) [reply airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/bad83517-5e54-4a3d-9b53-63e85fbd4d7c.json:4]: Yeah! stupid me!

File: docs/integrations/sources/clickhouse.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+| Logical Replication \(WAL\) | Coming soon |  |
+| SSL Support | No |  |
+| SSH Tunnel Connection | Coming soon |  |
+| Namespaces | Yes | Enabled by default |
+
+## Getting started
+
+### Requirements
+
+1. ClickHouse Server `21.3.10.1`.
```
Comment (davinchia) [inline docs/integrations/sources/clickhouse.md:29]: spoke about this: Let's make this version or later.

**Review by davinchia — APPROVED** [review]
NIce! Merge after the version is updated.

**Review by sherifnada — APPROVED** [review]
👏🏼

**Pull request merged**